pkgs/by-name/ol/ollama/package.nix

```diff
@@ -93,7 +93,7 @@ let
     ];
   };
 
-  metalFrameworks = with darwin.apple_sdk_11_0.frameworks; [
+  metalFrameworks = with darwin.apple_sdk.frameworks; [
```

Comment:
This is a noop on aarch64-darwin, therefore is not actually the fix.

Reply:
Thanks. You seem to know more about darwin than I do: do you know what the "right" package set to use would be? Version 11.0 seems not to be the best choice if there are other versions (such as 11.3) in use on real systems.

Reply:
11.0 is the best we have at the moment. (It'll be bumped to 11.3 soon, but that's just a minor release of 11.3.)

Reply:
(I'd be surprised if anyone is still running Big Sur in 2024? Maybe I'm wrong. 😅)